### PR TITLE
Fix for viewing offline and online players

### PR DESCRIPTION
**Pull request by Marcus101RR**
This is the final draft of fixing that annoying offline/online viewing of players and saving them. This doesn't fix the Invsee-ception issue.

### CODE REVIEW

File: InvSee/Commands.cs [hunk 1]
```diff
@@ -24,8 +24,10 @@ public static void DoInvSee(CommandArgs args)
 			{
 				bool restored = info.Restore(args.Player);
 
-				if (restored)
-					args.Player.PluginSuccessMessage("Restored your inventory.");
+				if (args.Player.Dead)
```
Comment (AxisKriel) [inline InvSee/Commands.cs:25]: If you're looking into disabling InvSee while the user is dead, then why perform this check after you've already run the `Restore` method? (two lines above). This check needs to go before that.

EDIT: I forgot which checks I performed in the restore method itself. If it already checks for this, and the sole purpose of this check is to provide a fine error message to the player, then it is acceptable.

File: InvSee/Commands.cs [hunk 2]
```diff
@@ -82,6 +92,7 @@ public static void DoInvSee(CommandArgs args)
 
 					PlayerData data;
 					string name = "";
+					int userid = 0;
```
Comment (Enerdy) [inline InvSee/Commands.cs]: `userid` doesn't seem to be used anywhere... I don't see the point in defining it. Is this a remnant of unfinished work (perhaps that other "invsee-ception" issue you've mentioned)?

**Comment by Marcus101RR** [comment]
I have fixed those issues that you found, and USER ID is being used in order to make sure the account is valid for offline Invsee as well as writing to the tsCharacter...

**Comment by AxisKriel** [comment]
I deleted my comment on the userID a minute later, but I guess github still sends email notifications. It was a huge oversight. It made sense after I noticed its use in the `UPDATE` query.

**Comment by Marcus101RR** [comment]
Oh, I was looking at my email, that explains why I couldn't find it on here.

**Comment by AxisKriel** [comment]
Alright, looks good.

**Pull request merged**